README.md

```diff
@@ -3,4 +3,4 @@ Demo repository for the tutorial about GitHub Pull Requests on [genicsblog.com](
 
 Please [read the tutorial](https://genicsblog.com/gouravkhunger/how-to-create-a-pull-request-in-github-correctly) before creating a pull request!
 
-- This line was added by [Gourav](https://github.com/gouravkhunger).
\ No newline at end of file
+- This line was added by [Ednei Lopes](https://github.com/Edy-ux).
```

Comment:
Hey, seems like you edited the first sentence whereas it would be better if you add a new one for your sentence. This way all the names would be preserved!